vendor/k8s.io/kubernetes/test/e2e/apimachinery/helpers.go

```diff
@@ -0,0 +1,359 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
```

Comment:
do you want to move it to `e2e/framework` dir?

Reply:
I would move it to test/e2e/framework/apihelper or something.